resources/views/shared/idea-card.blade.php

```diff
@@ -22,7 +22,8 @@
     </div>
     <div class="card-body">
         <p class="fs-6 fw-light text-muted">
-            {{$idea->content}}
+            <!-- TODO: Phan biet 2 cai duoi --> 
+            {{ $idea->content }}
```

Comment:
Nếu content có dạng như sau: 
<p>Here is new content </p><br><br> <div class="title">Hôm nay 18-12-2024 ...</div>